boards/arduino-mega2560/src/pins.rs

```diff
@@ -0,0 +1,346 @@
+        ///
+        /// * `AD7`: External memory interface
+        pub d29: porta::pa7::PA7,
+        /// `D30`
+        ///
+        /// * `AD15`: External memory interface
+        pub d30: portc::pc7::PC7,
+        /// `D31`
+        ///
+        /// * `AD14`: External memory interface
+        pub d31: portc::pc6::PC6,
+        /// `D32`
+        ///
+        /// * `AD13`: External memory interface
+        pub d32: portc::pc5::PC5,
+        /// `D33`
+        ///
+        /// * `AD12`: External memory interface
+        pub d33: portc::pc4::PC4,
+        /// `D34`
+        ///
+        /// * `AD11`: External memory interface
+        pub d34: portc::pc3::PC3,
+        /// `D35`
+        ///
+        /// * `AD10`: External memory interface
+        pub d35: portc::pc2::PC2,
+        /// `D36`
+        ///
+        /// * `AD9`: External memory interface
+        pub d36: portc::pc1::PC1,
+        /// `D37`
+        ///
+        /// * `AD8`: External memory interface
+        pub d37: portc::pc0::PC0,
+        /// `D38`
+        ///
+        /// * `T0`: Clock Input for Timer/Counter0
+        pub d38: portd::pd7::PD7,
+        /// `D39`
+        ///
+        /// * `ALE`: External memory Address Latch Enable
+        pub d39: portg::pg2::PG2,
+        /// `D40`
+        ///
+        /// `RD`: External memory Read Strobe
+        pub d40: portg::pg1::PG1,
+        /// `D41`
+        ///
+        /// `WR`: External memory Write Strobe
+        pub d41: portg::pg0::PG0,
+        /// `D42`
+        pub d42: portl::pl7::PL7,
+        /// `D43`
+        pub d43: portl::pl6::PL6,
+        /// `D44`
+        ///
+        /// * `OC5C`: Output Compare Channel `C` for Timer/Counter5
+        pub d44: portl::pl5::PL5,
+        /// `D45`
+        ///
+        /// * `OC5B`: Output Compare Channel `B` for Timer/Counter5
+        pub d45: portl::pl4::PL4,
+        /// `D46`
+        ///
+        /// * `OC5A`: Output Compare Channel `A` for Timer/Counter5
+        pub d46: portl::pl3::PL3,
+        /// `D47`
+        ///
+        /// * `T5`: Clock Input for Timer/Counter5
+        pub d47: portl::pl2::PL2,
+        /// `D48`
+        ///
+        /// * `ICP5`: Input Capture Trigger for Timer/Counter5
+        pub d48: portl::pl1::PL1,
+        /// `D49`
+        ///
+        /// * `ICP4`: Input Capture Trigger for Timer/Counter4
+        pub d49: portl::pl0::PL0,
+        /// `D50`
+        ///
+        /// * `MISO`: SPI bus Master In/Slave Out
+        /// * `PCINT3`: External Interrupt (Pin Change)
+        pub d50: portb::pb3::PB3,
+        /// `D51`
+        ///
+        /// * `MOSI`: SPI bus Master Out/Slave In
+        /// * `PCINT2`: External Interrupt (Pin Change)
+        pub d51: portb::pb2::PB2,
+        /// `D52`
+        ///
+        /// * `SCK`: SPI bus Serial Clock
+        /// * `PCINT1`: External Interrupt (Pin Change)
+        pub d52: portb::pb1::PB1,
+        /// `D53`
+        ///
+        /// * `SS`: SPI bus Chip Select
```

Comment:
```suggestion
        /// * `CS`: SPI bus Chip Select
```

Reply:
Well I tend to prefer the terminology "Chip Select" when possible, but since the datasheet uses "Slave Select" I've switched it to be consistent with that.